src/devices/machine/ram.h

```diff
@@ -59,32 +59,36 @@ class ram_device : public device_t
 	uint32_t size(void) const { return m_size; }
 	uint32_t mask(void) const { return m_size - 1; }
 	uint8_t *pointer(void) { return &m_pointer[0]; }
-	static uint32_t parse_string(const char *s);
 	uint32_t default_size(void) const;
-	const char *extra_options(void) const { return m_extra_options; }
+	const std::vector<uint32_t> &extra_options(void) const;
```

Comment:
Stray `(void)`?  This is a member function, it will never be valid in C (as opposed to C++), so the `void` isn't necessary to indicate it isn't a K&R function.